packages/um7/package.py

```diff
@@ -79,24 +112,18 @@ def install(self, spec, prefix):
         env["openmp"] = boolstr("~omp" in spec)
         env["netcdf"] = boolstr("~netcdf" in spec)
 
-        hg = 3  # build HadGEM3 ONLY here
-
         # Whether to build debug --jhan: adjust path to configs
         if opt_value == "debug":
-            bld_config = f"bld-dbg-hadgem{hg}-C2.cfg"
-            um_exe = f"um_hg{hg}_dbg.exe"
+            um_exe = f"um_hg{_hg}_dbg.exe"
         else:
-            bld_config = f"bld-hadgem{hg}-mct.cfg"
-            um_exe = f"um_hg{hg}.exe"
+            um_exe = f"um_hg{_hg}.exe"
 
-        bld_dir = f"ummodel_hg{hg}"
         # Build with fcm
-        fcm("build", "-f", "-j", "4",
-            join_path(bld_dir, "cfg", bld_config))
+        fcm("build", "-f", "-j", "4", _bld_cfg_path(opt_value, _hg))
 
         # Install
         mkdirp(prefix.bin)
         install(
-            join_path(bld_dir, "bin", um_exe),
+            join_path(f"ummodel_hg{_hg}", "bin", um_exe),
```

Comment:
Perhaps use a member function to create this string. `_bld_cfg_path` can then call the function too.

Reply:
Done.